src/redux/selectors.js

```diff
@@ -1,3 +1,23 @@
+import { isWithinRange } from 'date-fns';
+import { find } from 'lodash';
+
+const getVelocity = (stories, sprint) =>
+  stories.reduce((result, story) => {
+    if (isWithinRange(story.closedAt, sprint.startsAt, sprint.endsAt)) {
```

Comment:
I've run into some problems through with needing to align the `date-fns` methods so that they don't consider time when evaluating comparisons. This _may_ be another situation where we need to create a `isWithinDayRange` variant that aligns the inputs to start-of-day.

There are several examples of that before (and a helper called `alignToDay`) in the `src/util.js` file.

Reply:
On a related note, I just double-check that it's inclusive of the start and end parameters; which I think is the behavior we want. So we're good on that front.

Reply:
☝️ this is resolved in fb62324a9fa61c2e97978147078ef35e8906d431 now.